chrome.ts

```diff
@@ -238,7 +238,7 @@ class ChromeImpl implements Chrome {
         return;
       }
 
-      const folder = entry.folder;
+      const folder = entry.folder?.split('/').slice(1).join();
```

Comment:
Why is this change needed:question::thinking:

Reply:
This is needed for module resolution. (At least form my tests)

Reply:
Oh, I see. :slightly_smiling_face:  Maybe it's an OS specific bug. :cry: (This module is currently being tested only on Linux) What OS are you using?

Reply:
In any case, I think this bug should be fixed in a separate PR. Therefore, please let me revert for this change :wink: (I'll open an issue later)

Reply:
Windows